src/main/scala/cromwell/engine/backend/runtimeattributes/AttributeMap.scala

```diff
@@ -12,8 +12,16 @@ object AttributeMap {
   }
 }
 
-case class AttributeMap(attrs: Map[String, Seq[String]]) {
-  def get(key: RuntimeKey): Option[String] = attrs.get(key.key).flatMap(_.headOption)
+trait AttributeMapTrait {
```

Comment:
I see why it's called the way it is but it seems a little hokey. It seems like it'd be better for the concrete one below to explain what is special about it. OTOH I have a feeling this would get updated w/ the below one being called something like DefaultAttributeMap which would have likely caused me to complain as well so YMMV